ivadomed/scripts/prepare_dataset_vertebral_labeling.py

```diff
@@ -62,21 +62,20 @@ def extract_mid_slice_and_convert_coordinates_to_heatmaps(path, suffix, aim=-1):
     Returns:
         None. Images are saved in BIDS folder
     """
-    t = os.listdir(path)
-    t.remove('derivatives')
+    t = [d.name for d in Path(path).iterdir() if d.name != 'derivatives']
 
     for i in range(len(t)):
         sub = t[i]
```

Comment:
No sure why `t[i]` is assigned yet not used below. Is this `sub` the folder name which should be `subject`?